tests/installation/await_install.pm

```diff
@@ -109,7 +109,7 @@ sub _set_timeout {
 sub run {
     my $self = shift;
     # NET isos are slow to install
-    my $timeout = 2000;
+    my $timeout = 2400;
```

Comment:
this is too generic magic. based on the description what you want is to fix an issue in a particular aarch. Why not `my $timeout = is_s390x ? 2400 : 2000;` or something alike

Reply:
Good pointing, adding 400s directly is simple and doing no harm to other arches so I chose this way.

Reply:
Done, and the verify run is good: https://openqa.suse.de/tests/7273661#

Reply:
i want to point that in general is not a good idea. other than that no issue.